File: 1304-find-n-unique-integers-sum-up-to-zero/1304-find-n-unique-integers-sum-up-to-zero.py
class Solution:
    def sumZero(self, n: int) -> List[int]:
        val=int(n/2)
        lst=[]
        if(n%2==1):
            lst=list(range(-val,val+1))
        else:
            l1=list(range(-val,0))
            l2=list(range(1,val+1))
            lst=l1+l2
        return lst

File: 1304-find-n-unique-integers-sum-up-to-zero/1304-find-n-unique-integers-sum-up-to-zero.java
class Solution {
    public int[] sumZero(int n) {
        int ans[]=new int[n];
        int val=n/2;
        val*=(-1);
        if(n%2==0){
            for(int i=0; i<n; i++){
                if(val==0)val++;
                ans[i]=val;
                val++;
            }
        }else{
            for(int i=0; i<n; i++){
                ans[i]=val;
                val++;
            }
        }
        return ans;
    }
}